tests/pre_processing/test_extract_landscape.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from pre_processing.extract_landscape import get_facial_vectors
+import logging
+
+class TestExtractLandScape(unittest.TestCase):
```

Comment:
no shape_predictor_68_face_landmarks.dat is a precompiled neural net designed to pick out facial features